examples/all-clusters-app/esp32/main/main.cpp

```diff
@@ -339,7 +339,7 @@ class SetupListModel : public ListScreen::Model
         else if (i == 2)
         {
             app::Mdns::AdvertiseCommissionableNode();
-            OpenDefaultPairingWindow(ResetAdmins::kNo, PairingWindowAdvertisement::kMdns);
+            OpenDefaultPairingWindow(ResetAdmins::kYes, PairingWindowAdvertisement::kMdns);
```

Comment:
what does this change do and why is it required for ip provisioning?

Reply:
This is for testing - factory reset doesn't clear the admins right now (https://github.com/project-chip/connectedhomeip/issues/6475) so if you continuously put the M5 back into commissioning mode, memory eventually fills up. Since we're not handling multi-admin currently, we can just clear the past admins out when we go back into commissioning.

Reply:
Please create an *issue* and TODO of removing this `kYes`. This is critical. We do not want to show examples of setup where deleting everyone else is the default. That goes against core assumptions of Matter. Understood this is for testing, but it must not remain.